WalletWasabi/Legal/Assets/LegalDocuments.txt

```diff
@@ -177,7 +177,7 @@ I. TERMS AND CONDITIONS
     These Terms, together with any other terms incorporated into these Terms by reference and any other terms and conditions that apply to You, constitute the entire and exclusive agreement between us and You regarding its subject matter, and supersede and replace any previous or contemporaneous written or oral contract, warranty, representation or understanding regarding its subject matter. You acknowledge and agree that You shall have no remedies in respect of any statement, representation, assurance or warranty that is not set out in these Terms (or any other terms that are incorporated herein by reference).
 
 9.7 FORCE MAJEURE
-    Neither You nor we will be liable for delays in processing or other non-performance caused by such events as fires, telecommunications, utility, or power failures, equipment failures, labor strike, riots, war, nonperformance of our vendors or suppliers, acts of God, or other causes over which the respective party has no reasonable control; provided that the party has procedures reasonably suited to avoid the effects of such acts. The damages and consequences arising from these shall be borne by each party individually.
+    Neither You nor we will be liable for delays in processing or other non-performance caused by such events as fires, telecommunications, utility, or power failures, equipment failures, labor strike, riots, war, nonperformance of our vendors or suppliers, acts of God, or other causes over which the respective party has no reasonable control; provided that the party has procedures reasonably suited to avoid the effects of such events. The damages and consequences arising from these shall be borne by each party individually.
```

Comment:
GitHub shows that something is changed here, but I cannot tell what exactly...

Reply:
Before:
Neither You nor we will be liable for delays in processing or other non-performance caused by such events as fires, telecommunications, utility, or power failures, equipment failures, labor strike, riots, war, nonperformance of our vendors or suppliers, acts of God, or other causes over which the respective party has no reasonable control; provided that the party has procedures reasonably suited to avoid the effects of such **acts**

After:
Neither You nor we will be liable for delays in processing or other non-performance caused by such events as fires, telecommunications, utility, or power failures, equipment failures, labor strike, riots, war, nonperformance of our vendors or suppliers, acts of God, or other causes over which the respective party has no reasonable control; provided that the party has procedures reasonably suited to avoid the effects of such **events**.